Meter.c

```diff
@@ -345,8 +345,10 @@ static void GraphMeterMode_draw(Meter* this, int x, int y, int w) {
    }
    for (; i < nValues - 1; i += 2, k++) {
       int pix = GraphMeterMode_pixPerRow * GRAPH_HEIGHT;
-      int v1 = CLAMP((int) lround(data->values[i] * pix), 1, pix);
-      int v2 = CLAMP((int) lround(data->values[i + 1] * pix), 1, pix);
+      if (this->total < FLT_EPSILON)
```

Comment:
Nope.
https://bitbashing.io/comparing-floats.html
"[F]loat values aren’t equidistant! For values smaller than 1, FLT_EPSILON quickly becomes too large to be useful."
Also creates discontinuity at 1 as you clamp to 1 in the next line (meter would jump from huge value to 1).

Reply:
So what is the correct way? FLT_MIN does not work either. Should I use an arbitrary small constant like 0.0001?

Reply:
What about going with the suggestion from @fasterit from above and using `1`. To understand why this is quite reasonable we need to have a look at the usual values that these meters are displaying. As those values are usually some integer source and rarely ever scaled down we can assume that the smallest granularity we are likely to see is `1 unit` where unit is typically `Bytes/s` or `KiB/s`. With this in mind there'd be rarely any reason to go for anything below `1/10000` AKA `<1E-4`. And even if your smallest value was something like `1 B/s` you'd probably still don't want to make this the maximum for your scale. Instead for all intents and purposes is's more reasonable to start your scale at `1KiB/s` if the common value you'll be seeing is much larger than this; and thus err on the topic of discarding extremely small/unlikely values.

Reply:
Ok, I set it to 1. If we'll add meters for things that are smaller than 1, it can still be changed again.